browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -10367,35 +10369,6 @@ index 68e80682d1d0c003493e20a63d634aa8e0526a7d..67d6f7f1e370a537e1f539f58d555546
  
      virtual void setStatusText(WebKit::WebPageProxy*, const WTF::String&) { }
      virtual void mouseDidMoveOverElement(WebKit::WebPageProxy&, const WebKit::WebHitTestResultData&, OptionSet<WebKit::WebEvent::Modifier>, Object*) { }
-diff --git a/Source/WebKit/UIProcess/API/C/WKContextConfigurationRef.cpp b/Source/WebKit/UIProcess/API/C/WKContextConfigurationRef.cpp
-index ea48784147548a7ac6840445f225aef79dc825a4..e245546937667b82375f8bd778e5f41e5c61b818 100644
---- a/Source/WebKit/UIProcess/API/C/WKContextConfigurationRef.cpp
-+++ b/Source/WebKit/UIProcess/API/C/WKContextConfigurationRef.cpp
-@@ -145,17 +145,18 @@ void WKContextConfigurationSetIgnoreSynchronousMessagingTimeoutsForTesting(WKCon
-     toImpl(configuration)->setIgnoreSynchronousMessagingTimeoutsForTesting(ignore);
- }
- 
--WKArrayRef WKContextConfigurationCopyOverrideLanguages(WKContextConfigurationRef)
-+/* playwright revert fb205fb */
-+WKArrayRef WKContextConfigurationCopyOverrideLanguages(WKContextConfigurationRef configuration)
- {
--    // FIXME: Delete this function.
--    return toAPI(&API::Array::create().leakRef());
-+    return toAPI(&API::Array::createStringArray(toImpl(configuration)->overrideLanguages()).leakRef());
- }
- 
--void WKContextConfigurationSetOverrideLanguages(WKContextConfigurationRef, WKArrayRef)
-+void WKContextConfigurationSetOverrideLanguages(WKContextConfigurationRef configuration, WKArrayRef overrideLanguages)
- {
--    // Use +[WKWebView _setOverrideLanguagesForTesting:] instead.
--    // FIXME: Delete this function.
-+    toImpl(configuration)->setOverrideLanguages(toImpl(overrideLanguages)->toStringVector());
```

Comment:
This will break our locale tests, the upstream change doesn't actually restore the previous behavior so we should keep this code in our fork.

Reply:
This actually is not breaking the tests because we still call `processPool->configuration().setOverrideLanguages` internally even on Windows and WebProcessPool was [patched](https://github.com/q66/WebKit/commit/ec4c7e874e004be2fffcacdca77d36dc54304e6b#diff-7625e12c845787a874aee6306cbd9a28fa43e425c69b6a45321aeba0f0e347d3) previously. So it should be fine from the roll stand point, even though we cannot use this API after the upstream fix (I commented on the webkit bug upstream).